KDIC/AppDelegate.m

```diff
@@ -1,26 +1,13 @@
-//
-//  AppDelegate.m
-//  KDIC
-//
-//  Created by Colin Tremblay on 11/2/13.
-//  Copyright (c) 2013 Colin Tremblay. All rights reserved.
-//
-
 #import "AppDelegate.h"
-#import <Crashlytics/Crashlytics.h>
-#import <FlurrySDK/Flurry.h>
 #import <AVFoundation/AVFoundation.h>
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    NSString *strings_private = [[NSBundle mainBundle] pathForResource:@"strings_private" ofType:@"strings"];
-    NSDictionary *keysDict = [NSDictionary dictionaryWithContentsOfFile:strings_private];
-    
-    [Crashlytics startWithAPIKey:keysDict[@"CrashlyticsAPIKey"]];
     
-    [Flurry setCrashReportingEnabled:NO];
-    [Flurry startSession:keysDict[@"FlurrySession"]];
+    //tabcolor
+    UITabBarController *tabBarController = (UITabBarController *) self.window.rootViewController;
+    tabBarController.view.tintColor = [UIColor redColor];
```

Comment:
I'm pretty sure you can (and should) set this in the Storyboard. You have to set a manual string value. Let me know if you have any problems with this.